modules/hosted-che/partials/proc_adding-the-action-to-a-github-repository-workflow.adoc

```diff
@@ -10,7 +10,7 @@ This section describes how to integrate the Try in Web IDE GitHub action to a Gi
 .Prerequisites
 
 * A GitHub repository
-* A devfile in the root of the GitHub repository. See xref:providing-a-devfile_{context}[Providing a devfile].
+* A xref:end-user-guide:authoring-devfiles-version-2.adoc[devfile] in the root of the GitHub repository (see xref:providing-a-devfile_{context}[Providing a devfile])
```

Comment:
No need of article "a"

Reply:
this is not a bout a specific devfile so `a` is appropriate here